python/lsst/ctrl/oods/gen3ButlerIngester.py

```diff
@@ -286,8 +276,10 @@ def clean(self):
 
         butler = self.createButler()
 
+        await asyncio.sleep(0)
```

Comment:
Again, adding these in loops makes sense, but here maybe not, unless each of these steps takes a long time (which I don't think `refresh()` should).

Reply:
I've removed that. I'm more worried about the queryDatasets, queryCollections, and pruneDatasets taking a long time.